src/explore-education-statistics-common/src/services/publicationService.ts

```diff
@@ -79,11 +79,11 @@ export interface ReleaseNote {
 
 // eslint-disable-next-line no-shadow
 export enum ReleaseType {
-  AdHoc = 'Ad Hoc',
-  Experimental = 'Experimental',
+  AdHocStatistics = 'Ad Hoc Statistics',
```

Comment:
Searched the UI tests and it doesn't look like these changes would affect anything but just checking?

Reply:
Tests passed ok (other than known issues)

Reply:
I've made a few updates to the backend and frontend unit tests - not to fix tests but just to change test data from `Ad Hoc` to `Ad Hoc Statistics`